lms/djangoapps/courseware/date_summary.py

```diff
@@ -204,6 +205,9 @@ class VerifiedUpgradeDeadlineDate(DateSummary):
 
     @property
     def link(self):
+        ecommerce_service = EcommerceService()
+        if ecommerce_service.is_enabled(self.user):
+            return ecommerce_service.payment_page_url()
         return reverse('verify_student_upgrade_and_verify', args=(self.course.id,))
```

Comment:
Do we need a similar check [here](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/verify_student/views.py#L483)?